pkg/workflow/argo.go

```diff
@@ -730,10 +730,10 @@ func generateSubchartHelmRelease(a v1alpha1.Application, appName, scName, versio
 			TargetNamespace: targetNS,
 			Timeout:         a.Spec.Release.Timeout,
 			Install: &fluxhelmv2beta1.Install{
-				DisableWait: false,
+				DisableWait: a.Spec.Release.Install.DisableWait,
```

Comment:
Make sure to validate that these properties you are accessing aren't nil

Reply:
Ah yes - good catch. Let me add the getter function